go/tasks/pluginmachinery/utils/error_collection.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+type ErrorCollection struct {
```

Comment:
Can you use this? https://github.com/lyft/flytestdlib/blob/537f86093d86af270aab300742ee1a56f2905885/config/errors.go#L11

Reply:
i saw that, its not in the errors package but in config. isn't that weird?